yt_dlp/extractor/chzzk.py

```diff
@@ -0,0 +1,121 @@
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    traverse_obj,
+    parse_iso8601,
+)
+
+
+class CHZZKLiveIE(InfoExtractor):
+    IE_NAME = 'chzzk:live'
+    _VALID_URL = r'https?://chzzk\.naver\.com/live/(?P<id>[^/#?]+)'
+    _TESTS = [{
+        'url': 'https://chzzk.naver.com/live/c68b8ef525fb3d2fa146344d84991753',
+        'only_matching': True,
+    }]
+
+    def _real_extract(self, url):
+        channel_id = self._match_id(url)
+        live_detail_response = self._download_json(
+            f'https://api.chzzk.naver.com/service/v1/channels/{channel_id}/live-detail', channel_id,
+            note='Downloading channel info',
+            errnote='Unable to download channel info')
+        live_detail = live_detail_response.get('content')
+
+        if live_detail.get('status') == 'CLOSE':
+            raise ExtractorError('The channel is not currently live', expected=True)
+
+        live_playback = self._parse_json(live_detail.get('livePlaybackJson'), channel_id)
+
+        thumbnails = []
+        thumbnail_template = traverse_obj(live_playback, ('thumbnail', 'snapshotThumbnailTemplate'))
+        for width in traverse_obj(live_playback, ('thumbnail', 'types')):
+            thumbnails.append({
+                'id': width,
+                'url': thumbnail_template.replace('{type}', width),
+                'width': int(width),
+            })
+
+        formats, subtitles = [], {}
+        for media in live_playback.get('media'):
+            media_url = media.get('path')
+            fmts, subs = self._extract_m3u8_formats_and_subtitles(media_url, channel_id, 'mp4')
+            if media.get('mediaId') == 'LLHLS':
+                for fmt in fmts:
+                    fmt['format_id'] += '-ll'
+            formats.extend(fmts)
+            self._merge_subtitles(subtitles, subs)
```

Comment:
Need to actually be saved
```suggestion
            self._merge_subtitles(subs, target=subtitles)
```